.github/workflows/visual-regression-tests.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    types:
+      - ready_for_review
+
+jobs:
+  visual-regression-test-components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Check for cached Ruby gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: ${{ runner.os }}-gems-
+      - name: Get Yarn cache directory path
+        id: yarn-cache-directory-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Check for cached Yarn dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-directory-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-yarn-
+      - run: bundle install --jobs 4 --retry 3 --path vendor/bundle
```

Comment:
Any reason why you've gone for `--path vendor/bundle` over `--deployment`? the latter installs to vendor/bundle and also checks the Gemfile is up to date so is slightly preferable.

As I understand both warn as being deprecated as bundle now wants you to specify them in separate commands (which is a bit annoying - we'll need to update the docs to specify the newer approach)

Reply:
No reason - I took it from the [Ruby gem example](https://docs.publishing.service.gov.uk/manual/test-and-build-a-project-with-github-actions.html#a-ruby-gem) in the docs. I'll update it to deployment.

Reply:
Oh yes, sorry. Most gems can't use `--deployment` as they don't have their Gemfile.lock in the repo. This app is somewhat special in it's half gem half app state.